presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -411,6 +411,13 @@ protected final QueryRunner getQueryRunner()
         return queryRunner;
     }
 
+    protected final DistributedQueryRunner getDistributedQueryRunner()
```

Comment:
please update `(DistributedQueryRunner) getQueryRunner()` places to use the new method